views/tasks.py

```diff
@@ -18,13 +18,17 @@
 @app.route('/tasks/employees/load/s3', methods=['GET'])
```

Comment:
This methods are only supposed the be called via the cron job and not from a different client? 
Asking because the request timeout for regular calls is pretty low (I think I read something about 30s-60s) and we would run into timeouts.